.maintain/reset-spec/peregrine_stg_kilt.py

```diff
@@ -14,9 +14,9 @@ def update_spec(input: typing.Dict):
             "/dns4/eyrie-1.kilt.io/tcp/30345/p2p/12D3KooWR2wXmkcnQHZCLEpC9GJLtDJTYRoaMMcJrE6Xc8LioZtH",
         ],
         "chainType": "Live",
-        "name": "KILT Peregrine Stagenet",
+        "name": "KILT Peregrine-stg",
```

Comment:
AFAIK, this will "break" the custom styling of our Stagenet in the Polkadot Apps. So we need to make a PR to fix this.